packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -217,4 +217,8 @@ export default class HostingGeneralInformationsCtrl {
   isHostingOffer() {
     return !includes(['KIMSUFI_2015', '__60_FREE', 'DEMO_1_G', 'START_1_M', 'START_10_M', '_ASPFREE'], this.$scope.hosting.offer);
   }
+
+  goToPrivateSqlActivation() {
+    this.$state.go('app.hosting.private-sql-activation');
```

Comment:
```suggestion
    return this.$state.go('app.hosting.private-sql-activation');
```
so we can chain `goToPrivateSqlActivation` if we ever want to :)